api/v1alpha1/certificatestore_types.go

```diff
@@ -42,6 +42,7 @@ type CertificateStoreStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+// +kubebuilder:deprecatedversion:warning="v1alpha1 of the eraser API has been deprecated. Please migrate to v1beta1."
```

Comment:
I'm assuming this is auto generated? Curious what "eraser API" is...

Reply:
that's just copied from eraser project, I'll udpate it. Thanks for catching!